static/app/components/core/principles/motion/motion.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Motion
+layout: document
+description: Guidelines for using animation in Sentry's user interface.
+---
+
+import {MotionPlayground} from 'sentry/stories/playground/motion';
+
+Motion in the UI, when used thoughtfully, can bring interfaces to life and guide users through tasks. Effective animations provide feedback, reduce user frustration, and give a sense of direct manipulation of on-screen elements. They also make our product feel snappy and modern by conveying responsiveness. The following guidelines outline our motion design principles, best practices for using animation in common scenarios, and accessibility considerations.
+
+<MotionPlayground />
+
+# Principles
+
+## Purposeful
+
+Use animation only when it serves a clear purpose. Every motion should communicate something useful (e.g. feedback, a state change) and not exist just for decoration. If an animation doesn’t help the user achieve a goal or learn something, reconsider using it.
+
+## Responsive
+
+Keep animations fast and efficient so the interface feels snappy. Most UI animations should complete in under 600ms. Very short durations (around 100ms) are perceived as instantaneous—use these for simple micro-interactions so users never feel slowed down. Strike a balance: slow enough to be noticed and understood, but never so slow that the user is waiting on the animation.
```

Comment:
A little nit-picky, but I interpret "responsive" as "in response to something" (there's a connotation of speediness, but that isn't the first thing that comes to mind). The descriptions seem to be referring to snappiness, which is a good thing to mention, but if that's the intent then let's use a different, more direct title?